src/Data.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+namespace Ohanzee\Helper;
+
+class Data
+{
+    /**
+     *
+     * Regexp to validate string as Binary
+     * @var PCRE
+     *
+     */
+    const BINARY_REGEXP       = '/(?:[01]{8}\s?){0,12}/';
+    
+    /**
+     *
+     * Regexp to validate string as Timestamp
+     * @var PCRE
+     *
+     */
+    const TIMESTAMP_REGEXP    = '/\d{4}-\d{1,2}-\d{1,2}\s\d{2}:\d{2}:\d{2}/';
+    
+    /**
+     *
+     * Array of invalid Timestamp values
+     * @var array
+     *
+     */
+    private static $invalid_timestamps  = array(
```

Comment:
Sorry for the long delay. Been working a new job/busy during the holidays. Moved constant arrays to getter methods.